plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/PostgreSqlQueryRunner.java

```diff
@@ -55,6 +55,8 @@ public static DistributedQueryRunner createPostgreSqlQueryRunner(
             connectorProperties.putIfAbsent("connection-url", server.getJdbcUrl());
             connectorProperties.putIfAbsent("connection-user", server.getUser());
             connectorProperties.putIfAbsent("connection-password", server.getPassword());
+            connectorProperties.putIfAbsent("user-credential-name", "postgresql-user");
+            connectorProperties.putIfAbsent("password-credential-name", "postgresql-password");
```

Comment:
Hm, the only mention of these two properties I found is in the release notes for some older releases. Are they documented somewhere? They are a little tricky to wrap your head around, with the indirection and so on.